WordPress/wordpress-playground
# Export just wp-content instead of full WordPress

## Description 

This PR modifies the Playground import/export feature to focus on the wp-content directory instead of the entire WordPress installation.

* Exporting from Playground will only download the `wp-content` directory without Playground artifacts like the `sqlite-database-integration` plugin or the `0-playground` mu-plugin
* Importing into Playground will overwrite Playground files the imported files, including the `wp-includes`, `wp-admin`, and the `wp-content` directories. The `wp-content` directory gets a special treatment in that Playground artifacts such as the `sqlite-database-integration` and the `0-playground` mu-plugin are restored.

## Rationale

Exporting the full WordPress from Playground doesn't make much sense. It's an optimized bundle that's missing some files, it locks you in in an outdated WordPress version, and it inflates the zip size. What matters is the `wp-content` directory. This should enable easy GitHub-based management of WordPress core feature showcases.

## Remaining work

- [x] Add E2E tests to confirm that exporting and importing a zip file continue to work

## Other changes

* **Breaking: Removes the `replaceSite` step**. SourceGraph suggests it wasn't used in any public repo 🤞  Use the new `importWordPressFiles` step instead.
* Splits the import/export steps into different files

## Testing

* Verify export contains just the `wp-content` directory
* Test import works with zips containing just wp-content or wp-content + wp-includes
* Confirm the WordPress PR Previewer changes and TypeScript fixes work as expected
